Sources/Sentry/SentryTracer.m

```diff
@@ -105,16 +105,29 @@ - (void)finishWithStatus:(SentrySpanStatus)status
     [self captureTransaction];
 }
 
+- (nullable id<SentrySpan>)getLatestActiveSpan
+{
+    NSArray<id<SentrySpan>> *spans = [_spans copy];
+    if (spans.count > 0) {
+        for (int i = (int)spans.count - 1; i >= 0; i--) {
+            if (!spans[i].isFinished) {
+                return spans[i];
+            }
+        }
+    }
```

Comment:
Can you iterate a mutating `_span` safely in Obj-C?
In C# this would throw an exception

Reply:
I don't see how this would throw an exception in c#.
Unless you are talking about the lake of synchronization, I'm fixing this.

Reply:
I didn't notice the `copy` in there. 
It wouldn't even get to the iteration, it would crash on `copy` then since it's not synchronized.